tools/generators/lib/XCScheme/src/CreateTestAction.swift

```diff
@@ -104,6 +112,13 @@ buildConfiguration = "\#(buildConfiguration)"
             components.append(#"disablePerformanceAntipatternChecker = "YES""#)
         }
 
+        if let appLanguage {
+            components.append("language = \"\(appLanguage)\"")
+        }
+        if let appRegion {
+            components.append("region = \"\(appRegion)\"")
+        }
+
```

Comment:
Can you confirm this is the order that Xcode puts these values as well? Seems logical, being alphabetical, but it's something we have to check.

Reply:
looks like they are added to the end. I can rework this.

```
   <TestAction
      buildConfiguration = "Debug"
      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
      shouldUseLaunchSchemeArgsEnv = "YES"
      language = "en"
      region = "US">
```

Reply:
actually this looks correct, added to the end in alphabetical order.